sim/side.ts

```diff
@@ -262,6 +265,12 @@ export class Side {
 	hasAlly(pokemon: Pokemon) {
 		return pokemon.side === this || pokemon.side === this.allySide;
 	}
+	get subActives() {
+		return this.battle.gameType === 'rotation' ? this.pokemon.slice(1, 3) : [];
+	}
+	get activeAndSubActives() {
+		return this.active.concat(this.subActives);
+	}
```

Comment:
Getters are frowned upon by @Zarel

Reply:
Yeah, now that I've given it some thought, there isn't much of a point to make these `get`